src/sidebars/favourites/tearout.js

```diff
@@ -11,37 +11,53 @@
                         var dragElement = element[0],
                             tearElement = dragElement.parentNode.parentNode,
                             tileWidth = tearElement.clientWidth,
-                            tileHeight = tearElement.clientHeight;
+                            tileHeight = tearElement.clientHeight,
+                            store;
 
-                        function createConfig() {
+                        function createConfig(minWidth, minHeight, width, height, url, resizable, maximizable, showTaskbarIcon, saveWindowState) {
```

Comment:
If there's no sensible default, you wouldn't need to define one, since Openfin will apply its defaults. I'm fine with it being either way, just a long list of arguments to a function seems arbitrary without having to check the signature each time for the order.